actix-cors/src/builder.rs

```diff
@@ -115,10 +115,19 @@ impl Cors {
     /// `allowed_origin_fn` function is set, these functions will be used to determinate
     /// allowed origins.
     ///
-    /// Builder panics if supplied origin is not valid uri.
+    /// # Panics
+    ///
+    /// * If supplied origin is not valid uri, or
```

Comment:
Does it? Line 114 seems to suggest otherwise.

Reply:
Sorry I’m not following. How does it?

Reply:
Ahh, I'd forgotten how this works. So it assigns an error to the error field and then `finish` panics if there is an error from anywhere in the process. So in this way it's not *this* method that panics for invalid URLs. Therefore, it should not be documented here (in the same section as panics generated by this method, at least).

Thoughts on that?

Reply:
It was there before this PR. Should I remove it?

Reply:
Oh, sorry. Somehow didn't notice that. We can leave it in this PR but I'll likely move it later (probably with some other stuff).